src/platform/forms-system/src/js/definitions/address.js

```diff
@@ -258,9 +257,9 @@ export function uiSchema(
         if (isRequired) {
           const required = isRequired(formData, index);
           if (required && currentSchema.required.length === 0) {
-            currentSchema = _.set('required', requiredFields, currentSchema);
+            currentSchema = set('required', requiredFields, currentSchema);
           } else if (!required && currentSchema.required.length > 0) {
-            currentSchema = _.set('required', [], currentSchema);
+            currentSchema = set('required', [], currentSchema);
```

Comment:
```suggestion
            currentSchema.required = [];
```

Reply:
Same here, does it matter that this would mutate `addressSchema`?